app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
And when you `ember generate` or `ember destroy` reducers your `index.js` file is managed for you (like the `router.js` file does with routes).